packages/blade/src/components/Spinner/_decisions/decisions.md

```diff
@@ -0,0 +1,62 @@
+# Spinner <!-- omit in toc -->
+A spinner is an element with a looping animation that indicates loading is in process.
+
+<img src="./spinner-thumbnail.png" width="150" />
+
+## Design
+- [Figma - Spinner](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=14825%3A203592)
+
+
+## BaseSpinner
+A BaseSpinner will contain some extra props like `intent`. This will not be exposed to the consumer and will only be used for internal components.
+
+### API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| accessibilityLabel | `string` | `undefined` | The accessibility label (aria-label) for the spinner. | ✅ |
```

Comment:
Why is it that in `BaseSpinner` the **`accessibilityLabel`** is required but not required on Normal `Spinner`?

Reply:
Does that mean a `label` is always required?

Reply:
`accessibilityLabel` is only for a11y purposes not for actually showing some text label. Making it mandatory for BaseSpinner to encourage our internal developers (blade developers) to add an `accessibilityLabel` wherever they use the BaseSpinner but we don't yet want to enforce this on external consumers. 

Ref: https://github.com/razorpay/blade/pull/662#discussion_r955699523